source/fundamentals/connection.txt

```diff
@@ -55,10 +55,26 @@ options. In the example, we set two connection options:
 ``maxPoolSize=20`` and ``w=majority``. To learn more about connection
 options, read the :ref:`golang-connection-options` section of this guide.
 
-The following code shows how you can use a connection string in a
-client to connect to MongoDB. In this example, the client uses 
-the :ref:`Stable API <golang-stable-api>` feature, though
-you can connect to MongoDB without this feature.
+Create a Client to Connect to MongoDB
```

Comment:
Suggestion:
Maybe the information on this page could be organized as follows:

Connection URI
- Compose a Connection String
- Create a Client to Connect to MongoDB